cmd/configure/configure.go

```diff
@@ -55,7 +55,7 @@ func configureNode() error {
 
 	// Check systemctl first
 	if algod.IsService() {
-		if promptWrapperYes("Algorand is installed as a service. Do you wish to edit the service file to change the data directory? (y/n)") {
+		if promptWrapperYes("Algorand is installed as a service. Do you wish to edit the service file to change the data directory? (Y/N)") {
```

Comment:
```suggestion
		if promptWrapperYes("Algorand is installed as a service. Do you wish to edit the service file to change the data directory? (y/N)") {
```
Capitalization here indicates the default if the user presses enter without entering an explicit choice. The default here is No, so `y/N`

Reply:
This applies to all promptWrapperYes / promptWrapperNo invocations in this file